### PR TITLE
Fix fuzz 2

### PR DESCRIPTION
**Pull request by AdityaGarg8**
1002 was not applying using git apply and the problem turned out to be a fuzz error. Fixed it by repreparing the patch.

### CODE REVIEW

File: 1002-add-modalias-to-apple-bce.patch [hunk 1]
```diff
@@ -11,13 +11,15 @@ diff --git a/drivers/staging/apple-bce/apple_bce.c b/drivers/staging/apple-bce/a
...
-@@ -432,5 +432,6 @@ MODULE_LICENSE("GPL");
+@@ -434,5 +434,6 @@ MODULE_LICENSE("GPL");
  MODULE_AUTHOR("MrARM");
  MODULE_DESCRIPTION("Apple BCE Driver");
  MODULE_VERSION("0.01");
 +MODULE_ALIAS("pci:v0000106Bd00001801sv*sd*bc*sc*i*");
  module_init(apple_bce_module_init);
- module_exit(apple_bce_module_exit);
- -- 
+-module_exit(apple_bce_module_exit);
```
Comment (Redecorating) [inline 1002-add-modalias-to-apple-bce.patch:20]: The patch adding and removing this line seems unnecessary.

File: 1002-add-modalias-to-apple-bce.patch [hunk 2]
```diff
@@ -11,13 +11,15 @@ diff --git a/drivers/staging/apple-bce/apple_bce.c b/drivers/staging/apple-bce/a
...
+\ No newline at end of file
++module_exit(apple_bce_module_exit);
```
Comment (Redecorating) [inline 1002-add-modalias-to-apple-bce.patch:22]: see above comment

**Review by Redecorating — CHANGES_REQUESTED** [review]
Besides making it not add and remove the module_exit line, the PKGBUILD's shasum for the changed patch also needs updating. Could you also bump the pkgrel=1 to pkgrel=2, as I'd like to have that bumped when there are any changes to the PKGBUILD as the CI can overwrite releases.

File: 1002-add-modalias-to-apple-bce.patch (hunk 1 above)
Comment (AdityaGarg8) [reply 1002-add-modalias-to-apple-bce.patch:20]: But if I remove this, git feels bad :(
So, I let it be that way

File: 1002-add-modalias-to-apple-bce.patch (hunk 2 above)
Comment (AdityaGarg8) [reply 1002-add-modalias-to-apple-bce.patch:22]: But if I remove this, git feels bad :(
So, I let it be that way

File: 1002-add-modalias-to-apple-bce.patch (hunk 1 above)
Comment (Redecorating) [reply 1002-add-modalias-to-apple-bce.patch:20]: When the patch is as below, it applies for me. Did you change the length of the before and after hunks when you edited the patch? (the ,5 and ,6 in the line with @@)
```diff
From 153b587ed53135eaf244144f6f8bdd5a0fe6b69e Mon Sep 17 00:00:00 2001
From: Redecorating <69827514+Redecorating@users.noreply.github.com>
Date: Fri, 24 Dec 2021 18:12:25 +1100
Subject: [PATCH 1/1] add modalias to apple-bce

---
 drivers/staging/apple-bce/apple_bce.c     |  1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/staging/apple-bce/apple_bce.c b/drivers/staging/apple-bce/apple_bce.c
index a6a656f..8cfbd3f 100644
--- a/drivers/staging/apple-bce/apple_bce.c
+++ b/drivers/staging/apple-bce/apple_bce.c
@@ -432,5 +432,6 @@ MODULE_LICENSE("GPL");
 MODULE_AUTHOR("MrARM");
 MODULE_DESCRIPTION("Apple BCE Driver");
 MODULE_VERSION("0.01");
+MODULE_ALIAS("pci:v0000106Bd00001801sv*sd*bc*sc*i*");
 module_init(apple_bce_module_init);
 module_exit(apple_bce_module_exit);
\ No newline at end of file
--
2.34.1
```

Comment (AdityaGarg8) [reply 1002-add-modalias-to-apple-bce.patch]: check the latest commit

**Review by Redecorating — APPROVED** [review]
I forgot to come back and check that the CI finished ;-;

**Pull request merged**